config/crd/bases/operators.coreos.com_subscriptions.yaml

```diff
@@ -0,0 +1,2041 @@
+---
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Did this CRD accidentally get added? Upon a second look, not sure what it has to do w/ this PR

Reply:
OK, talked to Pete, not strictly related but going in w/ this PR anyways

Reply:
Nah it's good